<!DOCTYPE html>
<!--
	Transit by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Talks - Network of Mind</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-xlarge.css" />
		</noscript>
		<script type="text/javascript" src="js/jquery.min.js" ></script>
		<script type="text/javascript" src="js/skel.min.js" ></script>
		<script type="text/javascript" src="js/skel-layers.min.js" ></script>
		<script type="text/javascript" src="js/init.js" ></script>
	</head>
	<body>

		<!-- Header -->
			<header id="header">
				<h1><a href="index.html">Network of Mind</a></h1>
				<nav id="nav">
					<ul>
						<li><a href="index.html">Home</a></li>
						<li><a href="schedule.html">Talks</a></li>
						<li><a href="speakers.html">Speakers</a></li>
						<li><a href="https://docs.google.com/forms/d/e/1FAIpQLSfxncMo2lDZpimBC-_bj7VipcgXs_c3fBGnryKbcm1Utea5yw/viewform#start=openform" class="button special">Register now</a></li>
					</ul>
				</nav>
			</header>

		<!-- Main -->
			<section id="main" class="wrapper">
				<div class="container">

					<header class="major">
						<h2>Schedule</h2>
						<p>The talk schedule will be updated soon.</p>
					</header>

					<!-- Table -->
						<section>
							<h3>Talk Times</h3>
              <h4>Tuesday - 31st Jan</h4>
							<div class="table-wrapper">
								<table class="alt">
									<thead>
										<tr>
											<th width="15%">Speaker</th>
											<th width="63%">Abstract/Title</th>
											<th width="7%">Time</th>
                      <th width="5%">Room</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td></td>
											<td>Registration</td>
											<td>9:00-9:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
											<td>Opening</td>
											<td>9:30-9:45</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Max Bennett</b></td>
                      <td><b>Opening Lecture - Neuroscience Research</b> - While statistical analysis of a single network has received a lot of attention in recent years, with a focus on social networks, analysis of a sample of networks presents its own challenges which require a different set of analytic tools. Here we study the problem of classification of networks with labeled nodes, motivated by applications in neuroimaging. Brain networks are constructed from imaging data to represent functional connectivity between regions of the brain, and previous work has shown the potential of such networks to distinguish between various brain disorders, giving rise to a network (or graph) classification problem. Existing approaches to graph classification tend to either treat all edge weights as a long vector, ignoring the network structure, or focus on the graph topology while ignoring the edge weights.  Our goal here is to design a graph classification method that uses both the individual edge information and the network structure of the data in a computationally efficient way. We are also interested in obtaining a parsimonious and interpretable representation of differences in brain connectivity patterns between classes, which requires variable selection. We propose a graph classification method that uses edge weights as variables but incorporates the network nature of the data via penalties that promotes sparsity in the number of nodes. We implement the method via efficient convex optimization algorithms and show good performance on  data from two fMRI studies of schizophrenia.</td>
											<td>9:45-10:10</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Kewei Chen</b></td>
                      <td><b>Construction and validation of trajectories of the brain function and the time course of the neurodegeneration in the middle aged and elder cognitively unimpaired/impaired individuals</b> -Researches on normal and pathological aging in recent years provided clear evidence that the anomalous brain structural, functional changes appear years before clinical symptoms and/or observable deterring behavior/cognition. The utilization of state of art neuroimaging techniques, comprehensive statistical modeling, well-designed, integrated behavior/cognitive data acquisition and advanced machine learning is allowing the conceptual inception, qualitative realization, and quantitative characterization of a series of events, either pathological, biochemical, metabolic, structural or behavioral over a wide age range starting from the forth decade or even earlier. Using the most common dementia type, Alzheimer’s disease as an example, we introduce the neuroimaging techniques including fluorodeoxyglucose positron emission tomography (FDG-PET), amyloid PET, tau PET, and various MRI sequences as quantitative and sensitive biomarkers, the mathematical and statistical tools to quantify the event sequences. Part of this event sequencing analysis is to statistically examine if one event (the start of beta-amyloid built-up, e.g.) is earlier than another (the initiation of tau protein, e.g.). With proper linear or non-linear model established and the corresponding anomalous age onset (AAO) defined for each event, the hypothesis that AAO for one event is earlier than AAO for another can be examined statistically permitting us to construct the ordering of the events over normal and pathological aging process, forming the foundation for early intervention strategy for delay or even prevention of the disease. Part of the strategy is to improve the statistical power for what we proposed pre-symptomatic AD intervention trial in cognitively normal people at highest imminent risk of AD with these surrogate markers developed. In conclusion, the examination of various interactive aspects of the brain aging using neuroimaging and statistical means is important part of understanding the network of the mind theoretically and is also helpful practically to construct sensitive pre-symptomatic AD biomarkers for prevention trials such as “Alzheimer’s Prevention Initiative.”</td>
											<td>10:15-11:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Morning Tea (catered)</b></td>
											<td>11:00-11:30</td>
											<td></td>
										</tr>
									</tbody> 
									<tbody>
										<tr>
                      <td><b>Raquel	Prado</b></td>
                      <td><p><b>Bayesian models and methods for brain images and signals</b> - I will present some recent modeling approaches for detecting brain activation and connectivity from brain images and signals. I will discuss a Bayesian model for analyzing multi-subject functional magnetic resonance (fMRI) data from several brain regions. This model allows for simultaneous inference of connectivity networks and hemodynamic response functions that are region, task and subject-specific, while taking into account variations across subjects and experimental conditions. I will show how this approach was used for analyzing multi-subject fMRI data from a stroke study. I will then discuss models and computationally efficient tools for analyzingcomplex-valued fMRI (f(c)MRI) data. I will show how Bayesian variable selection approaches can lead to improved brain activation from f(c)MRI data. Finally, I will present a frequency-domain Bayesian hierarchical approach for spectral analysis of multiple related time series in the context of analyzing multi-channel electroencephalogram data.</p></td>
											<td>11:30-12:45</td>
											<td></td>
										</tr>
									</tbody> 
									<tbody>
										<tr>
                      <td><b>Veronica	Corona</b></td>
											<td>TBA</td>
											<td>12:15-12:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Lunch (catered)</b></td>
											<td>12:30-14:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Joana	Grah</b></td>
                      <td><b>Mathematical imaging – research in the Cambridge Image Analysis Group and specific applications in cancer research</b></td>
											<td>14:00-14:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Round table discussion: Lamiae Aziz</b></td>
											<td>14:30-16:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Public Event: Networks for big biomedical data</b></td>
											<td>18:30-20:00</td>
											<td></td>
										</tr>
									</tbody>
								</table>
							</div>

              <h4>Wednesday - 1st Feb</h4>
							<div class="table-wrapper">
								<table class="alt">
									<thead>
										<tr>
											<th width="15%">Speaker</th>
											<th width="63%">Abstract/Title</th>
											<th width="7%">Time</th>
                      <th width="5%">Room</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td></td>
											<td>Registration</td>
											<td>9:00-9:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Peter Robinson</b></td>
                      <td><b>Modelling of physical brain</b></td>
											<td>9:30-10:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Morning	tea	(catered)</b></td>
											<td>10:30-11:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Tomonobu Nakayama</b> and <b>Zdenka Kuncic</b></td>
                      <td><b>Neuromorphic switch networks: experiment meets theory</b></td>
											<td>11:00-12:15</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Rintaro Higuchi</b></td>
                      <td><b>Fluctuation analysis of polymer-coated Ag nanowire network</b></td>
											<td>12:15-12:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Lunch (catered)</b></td>
											<td>12:30-13:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Joe Lizier</b></td>
                      <td><b>Neural information processing</b></td>
											<td>14:00-14:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Round table dicussion Chair: Zdenka Kuncic</b></td>
											<td>14:30-16:00</td>
											<td></td>
										</tr>
									</tbody>
								</table>
							</div>

              <h4>Thursday - 2nd Feb</h4>
							<div class="table-wrapper">
								<table class="alt">
									<thead>
										<tr>
											<th width="15%">Speaker</th>
											<th width="63%">Abstract/Title</th>
											<th width="7%">Time</th>
                      <th width="5%">Room</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td></td>
											<td>Registration</td>
											<td>9:00-9:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Genevera Allen</b></td>
                      <td><b>Population inference for brain networks</b> - Many are interested in understanding how the brain communicates at a systems level and how these systems are disrupted in neurological conditions and diseases. By modeling the brain as a network of functional brain connections, we seek to find network patterns that are altered in a group of patients or are associated with symptom severity. To achieve this, we introduce new two-level models using Gaussian Graphical Models to describe subject-level networks and Generalized Linear Models to describe population-level effects as a function of subject-level network patterns. This problem leads to a new statistical paradigm which we term Population Inference post Model Selection (PIMS). To address this, we present a new estimation and inference technique, R^3, which employs resampling, randomized network selection, and random effects test statistics. Our method offers substantial improvements in statistical powerful and yields fewer false positives than existing approaches. We use our techniques to discover alterations in functional brain networks of patients with autism and neurofibromatosis.</td>
											<td>9:30-10:15</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Daniel Kessler</b></td>
                      <td><b>Neurodevelopment of Brain Connectivity: Challenges and Opportunities</b> - The human brain exhibits massive maturational change from childhood to adulthood. This presents a critical opportunity for psychiatry, as it is clear that many psychiatric disorders (ADHD, autism, schizophrenia) are fundamentally disorders of  neurodevelopment. Concurrently, there has been a paradigmatic shift in psychiatric imaging away from locationist approaches that aim to identify the spatial locus of dysfunction towards connectivist approaches that aim to understand the brain as a complex and coordinated network. In this talk, I present an emerging framework for characterizing and quantifying neurodevelopmental disturbances in the functional connectome. I focus on work from our group and others that characterizes normative developmental trajectories of the functional connectome and links dysmaturation to deleterious outcomes.</td>
											<td>10:15-11:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
											<td>Morning tea (catered)</td>
											<td>11:00-11:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Manuel Gomez Rodriguez</b></td>
                      <td><b>Learning and Forecasting Opinion Dynamics in Networks</b></td>
											<td>11:30-12:15</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Chen-Han Yu</b></td>
                      <td><b>Bayesian modeling of complex-valued fMRI signals.</b></td>
											<td>12:15-12:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Lunch (catered)</b></td>
											<td>12:30-14:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Soroosh Afyouni</b></td>
											<td>TBA</td>
											<td>14:00-14:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Round table discussion: Reproducible research in neuroimaging</b></td>
											<td>14:30-16:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Speakers Dinner</b></td>
											<td>TBA</td>
											<td></td>
										</tr>
									</tbody>
								</table>
							</div>

              <h4>Friday - 3rd Feb</h4>
							<div class="table-wrapper">
								<table class="alt">
									<thead>
										<tr>
											<th width="15%">Speaker</th>
											<th width="63%">Abstract/Title</th>
											<th width="7%">Time</th>
                      <th width="5%">Room</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td></td>
											<td>Registration</td>
											<td>9:00-9:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Alan Connelly</b></td>
                      <td><b>TBA</b></td>
											<td>9:30-10:15</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Thomas Carlson</b></td>
                      <td><b>The brain as its own decoder: predicting behavior from the geometrical structure of brain representations</b></td>
											<td>10:15-11:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Morning tea (catered)</b></td>
											<td>11:00-11:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Johann Gagnon-Bartsch</b></td>
                      <td><b>Removing unwanted variation with negative controls and replicates</b> - High-dimensional data, such as neuroimaging and genomics data, are plagued by unwanted variation -- systematic errors introduced by uncontrolled variations in experimental conditions. This unwanted variation is often stronger than the variation of interest, making analysis of the data challenging, and severely impeding the ability of researchers to capitalize on the promise of the technology.  One of the biggest challenges to removing unwanted variation is that the factors causing the variation are unmeasured or simply unknown. This makes the unwanted variation difficult to identify; the problem is essentially one of unobserved confounders. In my talk, I will discuss the use of negative controls and replicates to help solve this problem. A negative control is a variable known a priori to be unassociated with the factor of interest.  Replicates are repeated measurements of a single experimental unit. Both negative controls and replicates allow researchers to partly identify unwanted variation, and separate it from the biological variation of interest. I will present examples using diffusion tensor imaging data and microarray data.</td>
											<td>11:30-12:15</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Jesus Daniel Arroyo Relion</b></td>
                      <td><b>Network classification with applications to brain connectomics</b> - While statistical analysis of a single network has received a lot of attention in recent years, with a focus on social networks, analysis of a sample of networks presents its own challenges which require a different set of analytic tools.  Here we study the problem of classification of networks with labeled nodes, motivated by applications in neuroimaging. Brain networks are constructed from imaging data to represent functional connectivity between regions of the brain, and previous work has shown the potential of such networks to distinguish between various brain disorders, giving rise to a network (or graph) classification problem. Existing approaches to graph classification tend to either treat all edge weights as a long vector, ignoring the network structure, or focus on the graph topology while ignoring the edge weights. Our goal here is to design a graph classification method that uses both the individual edge information and the network structure of the data in a computationally efficient way. We are also interested in obtaining a parsimonious and interpretable representation of differences in brain connectivity patterns between classes, which requires variable selection. We propose a graph classification method that uses edge weights as variables but incorporates the network nature of the data via penalties that promotes sparsity in the number of nodes. We implement the method via efficient convex optimization algorithms and show good performance on  data from two fMRI studies of schizophrenia.</td>
											<td>12:15-12:30</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Lunch (catered)</b></td>
											<td>12:30-14:00</td>
											<td></td>
										</tr>
									</tbody>
									<tbody>
										<tr>
                      <td><b>Melanie Bahlo</b></td>
                      <td><b>Gene discovery in epilepsy</b></td>
											<td>14:00-14:30</td>
											<td>EAv 315</td>
										</tr>
									</tbody>
									<tbody>
										<tr>
											<td></td>
                      <td><b>Round table dicussion: Challenges in imaging omics with talk by Qiu Huang</b></td>
											<td>14:30-16:00</td>
											<td></td>
										</tr>
									</tbody>
								</table>
							</div>


						</section>
				</div>
			</section>

		<!-- Footer -->
			<footer id="footer">
				<div class="container">
					<div class="row">
						<div class="8u 12u$(medium)">
							<ul class="copyright">
								<li>&copy; Jean Yang. All rights reserved.</li>
							</ul>
						</div>
					</div>
				</div>
			</footer>

	</body>
</html>
